defaults/main.yml

```diff
@@ -192,3 +192,10 @@ rs_hooks_global_timeout: 300
 
 # Hostname
 hostname: example.org
+
+# Path to the executable hook scripts
+# see https://github.com/abrt/retrace-server/wiki/Hook-scripts
+rs_executable_hooks_path: "/usr/libexec/retrace-server/hooks/"
+
+# Global time limit for hook scripts (in seconds)
+rs_global_hook_timeout: 300
```

Comment:
These are already defined but the variable name in template is wrong.

`rs_hooks_executable_path`
https://github.com/abrt/ansible-role-retrace-server/blob/f9a1e30b20d0486ea8431454627348e43483cf8d/defaults/main.yml#L188

vs 

`rs_executable_hooks_path`
https://github.com/abrt/ansible-role-retrace-server/blob/f9a1e30b20d0486ea8431454627348e43483cf8d/templates/etc-retrace-server-hooks.conf.j2#L5